src/map/skill.cpp

```diff
@@ -9757,8 +9757,10 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 	case NPC_RANDOMMOVE:
 		if (md) {
 			md->next_walktime = tick - 1;
-			if (md->special_state.ai == AI_SPHERE)
+			if (md->special_state.ai == AI_SPHERE){
+				md->ud.walk_count = WALK_SKILL_INTERVAL - 1;
```

Comment:
Just curious why we are using the value 4 in this case.

Reply:
but I don't like it either

Reply:
Wouldnt ud->state.force_walk be the cleaner option?